docs/fromConfig/overrides.md

```diff
@@ -0,0 +1,39 @@
+---
+id: overrides
+title: Overriding any FromConfig
+---
+
+## Override any `FromConfig` instance
+
+This is the ultimate scape hatch, it allows overriding any `FromConfig` instances
+even ones defined inside an already defined `FromConfig` instance.
+
+## A motivating example
+
+Let's say we have a program that uses warp and we want to change slightly the
+way we get the port (we might want semantic names like `"http"` which would
+mean port `80`).
+
+That instance makes sense for the `Int` inside `warp` but not for `Int` in
+general so we can use the override mechanisms.
+
+So when we create the `Config` we use `mkConfig'` instead of `mkConfig`
+
+```haskell
+betterConfig = someConfig 
+    & addDefault "server.port" (overrideFetch @Int theNewFetchFunction)
```

Comment:
do you think it'd be helpful in this example to show an implementation of `theNewFetchFunction` ?

maybe it could be in another page like "Writing a `FromConfig` instance"

Reply:
Just as a log, I added the type and a reference to the future docs for writing a FromConfig instance since it's out of scope for this PR